build.zig

```diff
@@ -73,77 +160,49 @@ pub fn build(b: *std.Build) !void {
     raylib_parser_install.dependOn(&generateBindings_install.step);
 }
 
-// above: generate library
-// below: linking (use as dependency)
-
-fn current_file() []const u8 {
-    return @src().file;
-}
-
-const sep = std.fs.path.sep_str;
-const cwd = std.fs.path.dirname(current_file()).?;
-const dir_raylib = cwd ++ sep ++ "raylib" ++ sep ++ "src";
-
-const raylib_build = @import("raylib/src/build.zig");
-
-fn linkThisLibrary(b: *std.Build, target: std.Target.Query, optimize: std.builtin.Mode) *std.Build.Step.Compile {
-    const lib = b.addStaticLibrary(.{ .name = "raylib.zig", .target = b.resolveTargetQuery(target), .optimize = optimize });
-    lib.addIncludePath(.{ .path = dir_raylib });
-    lib.addIncludePath(.{ .path = cwd });
-    lib.linkLibC();
-    lib.addCSourceFile(.{ .file = .{ .path = cwd ++ sep ++ "marshal.c" }, .flags = &.{} });
-    std.log.info("include '{s}' to {s}", .{ dir_raylib, lib.name });
-    std.log.info("include '{s}' to {s}", .{ cwd, lib.name });
-    return lib;
-}
+fn emscriptenRunStep(b: *std.Build) !*std.Build.Step.Run {
```

Comment:
I snatched these emscripten utility functions from https://github.com/Not-Nik/raylib-zig/blob/devel/build.zig